src/test/kotlin/nl/info/zac/productaanvraag/ProductaanvraagServiceTest.kt

```diff
@@ -480,46 +480,56 @@ class ProductaanvraagServiceTest : BehaviorSpec({
                         communicatiekanaalNaam shouldBe "E-formulier"
                         bronorganisatie shouldBe "123443210"
                     }
-                    with(roleToBeCreated) {
-                        roleToBeCreated.size shouldBe 2
-                        with(roleToBeCreated[0]) {
-                            betrokkeneType shouldBe BetrokkeneTypeEnum.MEDEWERKER
-                            identificatienummer shouldBe fakeGebruikersnaamMedewerker.id
-                            roltype shouldBe rolTypeBehandelaar.url
-                            zaak shouldBe createdZaak.url
-                        }
-                        with(roleToBeCreated[1]) {
-                            betrokkeneType shouldBe BetrokkeneTypeEnum.NATUURLIJK_PERSOON
-                            identificatienummer shouldBe bsnNumber
-                            roltype shouldBe rolTypeInitiator.url
-                            zaak shouldBe createdZaak.url
-                        }
+                    roleToBeCreated.size shouldBe 2
```

Comment:
Multiple roles created in this collection, so this should be plural right?
```suggestion
                    rolesToBeCreated.size shouldBe 2
```